src/static.ts

```diff
@@ -513,8 +514,10 @@ type DropdownFormItem = BaseFormItem & {
   options?: Array<{
     value: string;
     label: string;
+    description?: string;
   }>;
   disabled?: boolean;
+  selectTooltip?: string;
```

Comment:
I think it would be better to give it a more generic name like `tooltip`. In the documentation you can say that initially we are only supporting it for select, however in the future we might want other fields to have tooltips as well

```suggestion
  tooltip?: string;
```

Reply:
Yea, its reference is the wrapper --> always show on top of title